CHANGELOG.rst

```diff
@@ -1,6 +1,133 @@
 Change Log
 ==========
 
+1.0.0: 2021/11/XX
+-----------------
+
+This the first version of `silx` supporting `PySide6` (for `Qt6`) and using `pytest` to run the tests.
+
+* `silx view`:
+
+  * Added Windows installer generation (PR #3548)
```

Comment:
This was already present before this version, so maybe "updated" is more appropriate.

Reply:
There was only a .zip file, no proper installer... even though the fat binary generation was already there (but I couldn't found its addition in the changelog).

Reply:
Ok, sounds good to me then.